### PR TITLE
Invoice payment docs update attempt2

### PR DESCRIPTION
**Pull request by KBbitsP**
Added JS snippets

### CODE REVIEW

**Review by reshmabidikar — COMMENTED** [review]
Shouldn't this PR be against the https://github.com/killbill/slate/tree/invoice-payment-docs-update branch?

File: source/includes/_invoice-payment.md [hunk 1]
```diff
@@ -131,12 +131,18 @@ KillBillClient::Model::InvoicePayment.find_by_id(payment_id,
...
+invoicePayment = invoicePaymentApi.get_invoice_payment(paymentId)
 ```
 
 ```javascript
+const invoicePaymentApi: killbill.InvoicePaymentApi = new killbill.InvoicePaymentApi(config);
+
+const paymentId = '79a9faee-29bc-43ac-ad7a-ad620d5f21cc';
+
+const invoicePayment: AxiosResponse<killbill.InvoicePayment,any> =
+      await invoicePaymentApi.getInvoicePayment(paymentId);
```
Comment (reshmabidikar) [inline source/includes/_invoice-payment.md:145]: is this formatting good?

File: source/includes/_invoice-payment.md [hunk 2]
```diff
@@ -258,17 +264,26 @@ KillBillClient::Model::InvoicePayment.refund(payment_id,
 ```python
 invoicePaymentApi = killbill.InvoicePaymentApi()
 
-payment_id = '8d85a8e8-c94b-438f-aac1-e8cb436b2c05'
+paymentId = '8d85a8e8-c94b-438f-aac1-e8cb436b2c05'
 
-transactionBody = killbill.PaymentTransaction(amount=50.0)
+transactionBody = killbill.InvoicePaymentTransaction(amount=50.0)
```
Comment (reshmabidikar) [inline source/includes/_invoice-payment.md:269]: I would suggest changing this to `transactionBody = InvoicePaymentTransaction(amount=50.0)` to keep it consistent with other Python code

File: source/includes/_invoice-payment.md [hunk 3]
```diff
@@ -349,17 +364,29 @@ KillBillClient::Model::InvoicePayment.chargeback(payment_id,
 ```python
 invoicePaymentApi = killbill.InvoicePaymentApi()
 
-payment_id = '2276b3c9-4e51-41b2-b5bf-9ddc11582ee4'
+paymentId = '2276b3c9-4e51-41b2-b5bf-9ddc11582ee4'
 
-transactionBody = killbil.PaymentTransaction(amount=50.0, currency='USD')
+transactionBody = killbill.InvoicePaymentTransaction(amount=50.0, currency='USD')
```
Comment (reshmabidikar) [inline source/includes/_invoice-payment.md:369]: Same comment as above, drop `killbill.`  here and in other places as applicable to keep the code consistent with other Python snippets.

**Review by reshmabidikar — COMMENTED** [review]
Looks good, I've left a few minor comments. Also, not sure I understand the Python code changes, these were already addressed in [this PR](https://github.com/killbill/slate/pull/260) I believe?

**Comment by KBbitsP** [comment]
Hi @reshmabidikar made the required changes.

**Comment by reshmabidikar** [comment]
> Also, not sure I understand the Python code changes, these were already addressed in https://github.com/killbill/slate/pull/260 I believe?

Could you please clarify this? Are these changes that were missed in the earlier PR?

**Review by reshmabidikar — APPROVED** [review]

**Review by reshmabidikar — APPROVED** [review]
This PR contains too many changes so approving it for now. We can revisit it once it is merged.

**Pull request merged**